srcpkgs/NetworkManager/template

```diff
@@ -1,9 +1,9 @@
 # Template file for 'NetworkManager'
-# when updating to 1.24 or newer, drop the compat
-# patch in gnome-control-center and rebuild it
+# when updating, drop 'nm122.patch' in gnome-control-center and rebuild it
+# if it builds, revbump it without the patch
```

Comment:
Do we need to do this?

Reply:
I'm not entirely sure, I think that message is @Gottox's doing. Personally I'd be fine with dropping that message because if we still need the patch after 8 minor versions we'll probably need it for a while longer.

Reply:
According to `git blame` it's @q66.

Reply:
?

the comment should be pretty clear, you're updating to networkmanager newer than 1.24, so you will need to remove the comment, remove the patch in gnome-control-center, and rebuild gnome-control-center.

Reply:
I attempted to rebuild gnome-control-center without the patch earlier and it failed. According to discussion above the relevant error was:

> /usr/bin/ld: /tmp/test-wifi-panel-text.ojS9Hb.ltrans0.ltrans.o: in function `nm_client_on_permission_change'

Reply:
you must have been doing something wrong, since the patch was added specifically for the reason our networkmanager is way out of date and the function was added as early as 1.24.x

Reply:
Alright, I've pushed a commit with a new version of gnome-control-center to this PR. I'll see if it builds correctly this time.

Reply:
the reason it's not building on CI is obvious - xbps-src pulling in the old nm when building gnome-control-center, not the new one

you will need to test it locally, and raise the gnome-control-center change as a separate PR after networkmanager is updated

Reply:
Ah, I was under the impression that the GH checks system "stacked" in such a way that it took into account changes from other commits in the same PR. I'll create a separate PR after this gets merged.

Reply:
@cinerea0 it kinda should have, I'm curious why it didn't...

Reply:
@ericonr Wait, are you saying that the builds should "stack" in the way I described? Also, now that everything has been mostly resolved, do you think it would be a good idea to merge this and then for me to create a separate PR for gnome-control-center?

Reply:
> Wait, are you saying that the builds should "stack" in the way I described?

Yes, they should. What's probably happening is what sgn commented below, that because there isn't a direct dependency chain, `gnome-control-center` gets built before `NetworkManager`, so it doesn't pick up the updated package. You can probably revbump `libnma` here to get "gcc" to also build, I think.